frontend/src/uxbox/main/ui/auth.cljs

```diff
@@ -0,0 +1,93 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; This Source Code Form is "Incompatible With Secondary Licenses", as
+;; defined by the Mozilla Public License, v. 2.0.
+;;
+;; Copyright (c) 2020 UXBOX Labs SL
+
+(ns uxbox.main.ui.auth
+  (:require
+   [cljs.spec.alpha :as s]
+   [beicon.core :as rx]
+   [rumext.alpha :as mf]
+   [uxbox.main.ui.icons :as i]
+   [uxbox.main.data.users :as du]
+   [uxbox.main.data.messages :as dm]
+   [uxbox.main.store :as st]
+   [uxbox.main.ui.messages :refer [messages]]
+   [uxbox.main.ui.auth.login :refer [login-page]]
+   [uxbox.main.ui.auth.recovery :refer [recovery-page]]
+   [uxbox.main.ui.auth.recovery-request :refer [recovery-request-page]]
+   [uxbox.main.ui.auth.register :refer [register-page]]
+   [uxbox.main.repo :as rp]
+   [uxbox.util.timers :as ts]
+   [uxbox.util.forms :as fm]
+   [uxbox.util.i18n :as i18n :refer [tr t]]
+   [uxbox.util.router :as rt]))
+
+(mf/defc goodbye-page
+  [{:keys [locale] :as props}]
+  [:div.goodbay
+   [:h1 (t locale "auth.goodbye-title")]])
+
+(mf/defc auth
+  [{:keys [route] :as props}]
+  (let [section (get-in route [:data :name])
+        locale (mf/deref i18n/locale)]
+    [:*
+     [:& messages]
+     [:div.auth
+      [:section.auth-sidebar
+       [:a.logo i/logo]
```

Comment:
Should have a href here, pointing to home for example.

Reply:
What is the home page in this example?

Reply:
I think the dashboard if the user is logged in, or the root if not. In fact, I think the / address should always redirect to dashboard when the user is logged in.

Reply:
But this is not an issue of this MR. What I'd do here is that the logo directs to /, and then see if we want this to go dashboard in other issue.

Reply:
ok